### ISSUE TITLE
Api are changing location

### ISSUE THREAD

**Issue by hfurubotten** [hfurubotten/enturclient#5]
The graphql API are changing domain and will be located on this url instead: 
`https://api.entur.io/journey-planner/v2/graphql`

- [x] Update resource URL
- [x] Research rate limiting on the new service urls

**Comment by hfurubotten** [comment]
Sent support ticket to Entur, and they have now added rate limiting information in the headers on the new endpoints as well. 

The rate limits seems to have been simplified to a single unit of measure, with 100 requests allowed pr. minute. 

Example:
```
Rate-Limit-Allowed → 100
Rate-Limit-Available → 99
Rate-Limit-Range → per-minute
Rate-Limit-Used → 1
```

**Comment by hfurubotten** [comment]
New endpoints implemented in v0.2.0
Rate limiting needs to be transferred to new issue about implementing throttling from it.
